src/utilities/theme/colorAdjustments.json

```diff
@@ -0,0 +1,546 @@
+    "baseColor": "onSurface",
+    "light": {"lightness": 88},
+    "dark": {"lightness": 88}
+  },
+  "iconSubduedOnLight": {
+    "baseColor": "onSurface",
+    "light": {"lightness": 43},
+    "dark": {"lightness": 43}
+  },
+  "textOnDark": {
+    "baseColor": "onSurface",
+    "light": {"lightness": 100},
+    "dark": {"lightness": 100}
+  },
+  "textOnLight": {
+    "baseColor": "onSurface",
+    "light": {"lightness": 13},
+    "dark": {"lightness": 13}
+  },
+  "textDisabledOnDark": {
+    "baseColor": "onSurface",
+    "light": {"lightness": 80},
+    "dark": {"lightness": 80}
+  },
+  "textDisabledOnLight": {
+    "baseColor": "onSurface",
+    "light": {"lightness": 63},
+    "dark": {"lightness": 63}
+  },
+  "textSubduedOnDark": {
+    "baseColor": "onSurface",
+    "light": {"lightness": 90},
+    "dark": {"lightness": 90}
+  },
+  "textSubduedOnLight": {
+    "baseColor": "onSurface",
+    "light": {"lightness": 38},
+    "dark": {"lightness": 38}
+  },
+  "interactive": {"baseColor": "interactive", "light": {}, "dark": {}},
+  "interactiveAction": {
+    "baseColor": "interactive",
+    "light": {"lightness": 44},
+    "dark": {"lightness": 56}
+  },
+  "interactiveActionDisabled": {
+    "baseColor": "interactive",
+    "light": {"lightness": 58},
+    "dark": {"lightness": 42}
+  },
+  "interactiveActionHovered": {
+    "baseColor": "interactive",
+    "light": {"lightness": 37},
+    "dark": {"lightness": 63}
+  },
+  "interactiveActionSubdued": {
+    "baseColor": "interactive",
+    "light": {"lightness": 51},
+    "dark": {"lightness": 49}
+  },
+  "interactiveActionPressed": {
+    "baseColor": "interactive",
+    "light": {"lightness": 31},
+    "dark": {"lightness": 69}
+  },
+  "interactiveFocus": {
+    "baseColor": "interactive",
+    "light": {"lightness": 58},
+    "dark": {"lightness": 42}
+  },
+  "interactiveSelected": {
+    "baseColor": "interactive",
+    "light": {"lightness": 96},
+    "dark": {"lightness": 4}
+  },
+  "interactiveSelectedHovered": {
+    "baseColor": "interactive",
+    "light": {"lightness": 89},
+    "dark": {"lightness": 11}
+  },
+  "interactiveSelectedPressed": {
+    "baseColor": "interactive",
+    "light": {"lightness": 82},
+    "dark": {"lightness": 18}
+  },
+  "neutral": {"baseColor": "neutral", "light": {}, "dark": {}},
+  "neutralActionDisabled": {
+    "baseColor": "neutral",
+    "light": {"lightness": 94},
+    "dark": {"lightness": 13}
+  },
+  "neutralAction": {
+    "baseColor": "neutral",
+    "light": {"lightness": 92},
+    "dark": {"lightness": 22}
+  },
+  "neutralActionHovered": {
+    "baseColor": "neutral",
+    "light": {"lightness": 86},
+    "dark": {"lightness": 29}
+  },
+  "neutralActionPressed": {
+    "baseColor": "neutral",
+    "light": {"lightness": 76},
+    "dark": {"lightness": 39}
+  },
+  "branded": {"baseColor": "branded", "light": {}, "dark": {}},
+  "brandedAction": {
+    "baseColor": "branded",
+    "light": {"lightness": 25},
```

Comment:
Just FYI this `25` lightness comes from manually pulling the lightness from the base jade color. In HSLuv, the lightness is `47.3`.

Reply:
Ah, that's interesting and weird. I stand corrected. Will read up on the color spaces and pick up the discussion next week!